backend/api/admin_views.py

```
@@ -0,0 +1,204 @@
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework import status
+from rest_framework.permissions import IsAuthenticated
+from django.shortcuts import get_object_or_404
+from firebase_admin import auth as firebase_auth
+from firebase_admin import firestore
+from django.db.models import Count, Sum, Avg
```

Ruff (0.8.2)

`django.db.models.Avg` imported but unused

Remove unused import: `django.db.models.Avg`

(F401)